.github/dependabot.yml

```diff
@@ -0,0 +1,10 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "monthly"
```

Comment:
Added a dependabot config as I noticed a few actions pretty out-of-date. This setting will make it open PRs monthly to update them. In MNE-Python we use weekly but figured you might want monthly here?

Reply:
yes, monthly is fine! 

thanks for the upgrades ;)